debian/control

```diff
@@ -13,7 +13,7 @@ Build-Depends: automake,
                cpio,
                cryptsetup-bin | cryptsetup,
                cython3,
-               debhelper (>= 9),
+               debhelper (>= 9.20160709) | dh-systemd,
```

Comment:
how about just bump the version to 
```
debhelper (>= 10),
```
? because 

- stretch comes with debhelper 10: https://packages.debian.org/stretch/debhelper
- buster comes with debhelper 12.1.1, see  https://packages.debian.org/buster/debhelper


so far, we only built ceph for buster and bullseye. see https://github.com/ceph/ceph-build/pull/1902/files#diff-ac9c3d44df3ef56e6433aecfc057f89efe44a9cd6335f21caf9c7b03d33fe47fR76

Reply:
Sure, fair enough.  Done.